src/legacy/core_plugins/data/public/search/aggs/buckets/_terms_other_bucket_helper.ts

```diff
@@ -55,18 +63,20 @@ const getAggResultBuckets = (aggConfigs, response, aggWithOtherBucket, key) => {
       // If you have multi aggs, we cannot just assume the first one is the `other` bucket,
       // so we need to loop over each agg until we find it.
       for (let aggId = 0; aggId < responseAggs.length; aggId++) {
-        const agg = responseAggs[aggId];
+        const aggById: any = responseAggs[aggId];
```

Comment:
IAggConfig or maybe even have a type for response aggs @lukeelmers ?

Reply:
Can we guarantee that the response will have IAggConfig as values?

Reply:
`IAggConfig` won't work here since this is part of the raw response.

Example output of `aggById`:
```js
{
  "doc_count_error_upper_bound": 0,
  "sum_other_doc_count": 96,
  "buckets": [
    {
      "3": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 51,
        "buckets": [
          {
            "key": "win 7",
            "doc_count": 25
          },
          {
            "key": "win xp",
            "doc_count": 22
          }
        ]
      },
      "key": "",
      "doc_count": 98
    },
    ...etc
  ]
}
```

AFAIK we do not have a proper type for ES responses, but would probably be useful to create eventually. In the meantime `Record<string, any>` would be at least a little more accurate.